l2geth/core/vm/interpreter.go

```diff
@@ -116,10 +121,6 @@ func NewEVMInterpreter(evm *EVM, cfg Config) *EVMInterpreter {
 				log.Error("EIP activation failed", "eip", eip, "error", err)
 			}
 		}
-		// Enable minimal eip 2929
-		if rcfg.UsingOVM {
-			enableMinimal2929(&jt)
```

Comment:
So previously we've always been enabling this. Please remind me again why it's now limited to Istanbul.

Reply:
We are currently always using the istanbul rule set. Placing the `enableMinimal2929` in that block is more clear of the functionality and ensures that we skip it when running the berlin rule set